src/Main.cpp

```diff
@@ -0,0 +1,54 @@
+/*
+    Main.cpp
+
+    Part of MR Lab Control Software
+
+    SPDX-License-Identifier: EUPL-1.2
+    SPDX-FileCopyrightText: Copyright (c) 2025 Martin Rumori/sonible GmbH
+ */
+
+#include <juce_gui_basics/juce_gui_basics.h>
+
+#include "controller/MainController.h"
+#include "view/MainWindow.h"
+
+namespace mrlab
+{
+
+//==============================================================================
+class MrLabControlApplication : public juce::JUCEApplication
+{
+public:
+    //==============================================================================
+    const juce::String getApplicationName() override { return JUCE_APPLICATION_NAME_STRING; }
+    const juce::String getApplicationVersion() override { return JUCE_APPLICATION_VERSION_STRING; }
+    bool moreThanOneInstanceAllowed() override { return false; }
+
+    //==============================================================================
+    void initialise (const juce::String& /*commandLine*/) override
+    {
+        mainController = std::make_unique<controller::MainController>();
+        mainWindow = std::make_unique<view::MainWindow> (getApplicationName(), *mainController);
+    }
+
+    //==============================================================================
+    void shutdown() override
+    {
+        mainWindow.reset();
+        mainController.reset();
```

Comment:
are these explicit resets really needed?

Reply:
No, they aren't, and maybe they will never be (as long as the declaration order of the members remains correct). I would still leave them in here for the moment, pretty sure the shutdown code is going to be refined in some way.